articles/quickstarts/qrng.md

```diff
@@ -48,16 +61,33 @@ We can use this representation to visualize what the code is doing:
 
 <img src="~/media/qrng-H.png" width="450" alt="Preparing a qubit in superposition">
 
-
 * Then we measure the qubit and save the output:
 
 <img src="~/media/qrng-meas.png" width="450" alt="Measuring a qubit and saving the output">
 
 Since the outcome of the measurement is completely random, we have obtained a random bit. We can call this operation several times to create integers. For example, if we call the operation three times to obtain three random bits, we can build random 3-bit numbers (that is, a random number between 0 and 7).
 
-## Creating a complete random number generator using a host program
 
-Now that we have a Q# operation that generates random bits, we can use it to build a complete quantum random number generator with a host program.
+## Creating a complete random number generator
+
+Now that we have a Q# operation that generates random bits, we can use it to build a complete quantum random number generator. We cans use the Q# standalone executable or use a host program. 
```

Comment:
```suggestion
Now that we have a Q# operation that generates random bits, we can use it to build a complete quantum random number generator. We can use the Q# standalone executable or use a host program. 
```

Reply:
Because currently we are showing the examples in all these options. Do you think we should remove the code samples with the host program options? I think it is good that we show them side by side once.